template_feed/Microsoft.DotNet.Web.ProjectTemplates.1.x/content/StarterWeb-CSharp/.template.config/template.json

```diff
@@ -106,6 +106,42 @@
       "defaultValue": "None",
       "description": "The type of authentication to use"
     },
+    "EFProvider": {
+      "type": "parameter",
+      "datatype": "choice",
+      "choices": [
+        {
+          "choice": "InMemory",
+          "description": "This database provider allows Entity Framework Core to be used with an in-memory database. This is useful when testing code that uses Entity Framework Core. The provider is maintained as part of the EntityFramework GitHub project."
+        },
+        {
+          "choice": "SqlServer",
+          "description": "This database provider allows Entity Framework Core to be used with Microsoft SQL Server (including SQL Azure). The provider is maintained as part of the EntityFramework GitHub project."
+        },
+        {
+          "choice": "SQLite",
+          "description": "This database provider allows Entity Framework Core to be used with SQLite. The provider is maintained as part of the EntityFramework GitHub project."
+        },
+        {
+          "choice": "MySql",
+          "description": "This database provider allows Entity Framework Core to be used with MySQL. The provider is maintained as part of the MySQL project."
+        },
+        {
+          "choice": "PostgreSQL",
+          "description": "This database provider allows Entity Framework Core to be used with PostgreSQL. The provider is maintained as part of the Npgsql project."
+        },
+        {
+          "choice": "PomeloMySql",
+          "description": "This database provider allows Entity Framework Core to be used with MySQL. The provider is maintained as part of the Pomelo Foundation Project."
+        },
+        {
+          "choice": "SapientGuardian",
+          "description": "This database provider allows Entity Framework Core to be used with MySQL. The provider is maintained as part of the Sapient Guardian Project."
+        }
+      ],
+      "defaultValue": "InMemory",
```

Comment:
Feels like SQLite might be a better default since it...
1. Actually persists data
2. Doesn't require a database server
3. Runs everywhere